.github/workflows/analyze-project.yaml

```diff
@@ -57,8 +57,8 @@ jobs:
           : Set Up Environment 🔧
           if (( ${+RUNNER_DEBUG} )) setopt XTRACE
 
-          print '::group::Enable Xcode 16.1'
-          sudo xcode-select --switch /Applications/Xcode_16.1.0.app/Contents/Developer
+          print '::group::Enable Xcode 16.4'
+          sudo xcode-select --switch /Applications/Xcode_16.4.app/Contents/Developer
```

Comment:
This appears to no longer be required, strictly speaking, since Xcode 16.4 is now the default on macos-15 runners as of last week.